draft-ietf-httpbis-resumable-upload.md

```diff
@@ -522,4 +522,6 @@ However, from experience with the tus project, users are often asking for a way
 
 This document is based on an Internet-Draft specification written by Jiten Mehta, Stefan Matsson, and the authors of this document.
 
+The Initial version of this document was inspired by the [tus v1 protocol](https://tus.io/) which provides a specification and working implementations for a resumable file upload protocol over HTTP. Memebers of its community helped significantly in creating the early Internet-Draft.
```

Comment:
Possible tweak if you're after an alternative, up to you if you want to pick it up

```suggestion
The [tus v1 protocol](https://tus.io/) is a specification for a resumable file upload protocol over HTTP. It inspired inspired the early design of this protocol. Members of the tus community helped significantly in the process of bringing this work to the IETF.
```